main/src/modules/reactiveLayer/behaviorManager/include/greeting.h

```diff
@@ -3,15 +3,30 @@
 #include <iomanip>
 #include <yarp/os/all.h>
 #include <yarp/sig/all.h>
-#include <map>
 
+#include <yarp/math/SVD.h>
+#include <yarp/math/Rand.h>
+#include "wrdac/clients/icubClient.h"
```

Comment:
Please re-order the includes from general (e.g. iostream) to more and more specific (first yarp, then wysiwyd includes etc.)